recipes/apdl-mode

```diff
@@ -0,0 +1,4 @@
+(apdl-mode
+:fetcher github
+:repo "dieter-wilhelm/apdl-mode"
+:files ("*.el" "*.org" "apdl-mode-readme.txt" ("doc" "doc/*.org" "doc/example.*") "matlib" "template"))
```

Comment:
I see your recipe is including files like TODO.org, NEWS.org, tutorial files, etc.  By default, MELPA doesn't package these kinds of files, and probably this package shouldn't either.  I think you can try to approximate the default recipe:
```suggestion
:files (:defaults "matlib" "template"))
```

Reply:
Hello @riscy 

Thanky ou for your suggestion, I applied it. The *.org files are documentation and example files.  But I think I have to rethink my documentation strategy anyway.  Maybe I can do with info files..

Reply:
If they're an intrinsic part of the package, it's fine to explicitly include them.  But it's worth bearing in mind for example that READMEs are excluded by default (and I think a TODO file at least is in this same camp).